wasm/jit/mmap.go

```diff
@@ -0,0 +1,19 @@
+package jit
+
+import "syscall"
+
+// Copy the code into the executable region
+// and returns the byte slice of the region.
+func mmapCodeSegment(code []byte) ([]byte, error) {
```

Comment:
is this testable, or does it create side effects? If untestable due to side effects, comment that?